src/core/connection/SSDConverter.hpp

```diff
@@ -0,0 +1,211 @@
+#pragma once
```

Comment:
Just wonder why it's in the header, 
One doesn't want to recompile all the dependents when a single comma is changed.

Reply:
just don't want to pollute the macro names

Reply:
You may use Macros in cpp and #undef them as well.

Reply:
Roger that. Will change now.